packages/Babylonian-Core.package/BPMethodExample.class/instance/asPragma.st

```diff
@@ -1,8 +1,9 @@
 serialization
 asPragma
-
-	^ '<exampleNamed: ''{1}''{2}{3}{4}>' format: {
+	
+	^ '<exampleNamed: ''{1}''{2}{3}{4}{5}>' format: {
 		self exampleName.
+		' expectedResult: ' , self expectedResult printString.
```

Comment:
This makes expectedResult a mandatory keyword for all examples. It would be nicer to keep it optional.